projects/packages/jetpack-mu-wpcom/src/utils.php

```diff
@@ -154,3 +154,30 @@ function get_wpcom_blog_id() {
 
 	return false;
 }
+
+/**
+ * Check if the site is a WordPress.com Atomic site.
+ *
+ * @return bool
+ */
+function is_woa_site() {
+	if ( ! class_exists( 'Automattic\Jetpack\Status\Host' ) ) {
+		return false;
+	}
+	$host = new Automattic\Jetpack\Status\Host();
+	return $host->is_woa_site();
+}
+
+/**
+ * Whether the current user is connected to WordPress.com.
+ *
+ * @param int $user_id the user identifier. Default is the current user.
+ * @return bool Boolean is the user connected?
+ */
+function is_current_user_connected( $user_id ) {
```

Comment:
The function name is misleading, if `$user_id` is given then it's not the current user.

Maybe `is_user_connected( $user_id )` ?